server/routes/problems.js

```diff
@@ -301,10 +301,14 @@ const postPublicationToProblemAndStage = async (req, res) => {
     //   await db.insertUserNotification(usersToNotify[i], publications[0]);
     // }
   }
-
-  resources.unshift(
-    (await db.insertResource(req.files[0].mimetype, req.files[0].location))[0],
-  );
+  if (req.files.length > 0) {
```

Comment:
Shouldn’t there always be a file - just in the case where only the editor was used one that was created by the front end from scratch?